### DISCUSSION TITLE
Creation issue from another or EPIC

### DISCUSSION THREAD

**Discussion by Alexis-PA** [community/community#128206] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Hello ✋ , I hope this is the good place to talk about this topic. 

When we create a check list in a issue description we have the possibility to convert this point into another issue. This is really a good thing when we use EPIC to list issue that they compose it. 

But when we do this (convert from the list), we don't have our template. 

It could be awesome to define a default template that it apply to this feature, don't you think ? 

Again, I hope this is the good place to talk about this and if it's not feel free to tell me where is the good place I'll copy my message. 

Thanks 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by dev-ugurkontel** [comment]
You are definitely in the right place regarding this topic! Using a checklist in descriptions and converting items into other issues is useful for managing subtasks, especially in EPICs. However, not using the default template during conversion can be annoying.

As you mentioned, it would be great to assign a default template to this feature. Although GitHub does not currently support this directly, there are workarounds:

Creating a Default Template:

Create a default template that will be useful for your common problems. This template can include title, tags, assignee, and description sections.
Save this template somewhere for easy access (for example, a note-taking app or a markdown file).
Copying Template When Converting:

When converting a list item to another issue, copy your default template and paste it into the description of the new issue.
Using GitHub Actions (more advanced):

For a more automated solution, you can set up a GitHub Actions workflow. This workflow can automatically apply the default template to converted issues. This requires some coding knowledge.
One of these suggestions should help you.

Additionally, some community-created plugins may offer the ability to apply templates to converted issues. You can discover them by searching "[marketplace github]" on GitHub Marketplace.

I hope this information helps! If you have any questions or other suggestions, don't hesitate.
